src/api-docs.js

```diff
@@ -1290,6 +1290,101 @@ const apiDoc = {
         },
       },
     },
+    '/wallet/p2sh/tx-proposal/mint-tokens': {
+      post: {
+        summary: 'Get the hex representation of a mint tokens transaction without input data.',
+        parameters: [
+          {
+            name: 'x-wallet-id',
+            in: 'header',
+            description: 'Define the key of the corresponding wallet it will be executed the request.',
+            required: true,
+            schema: {
+              type: 'string',
+            },
+          }
+        ],
+        requestBody: {
+          description: 'Data to mint tokens.',
+          required: true,
+          content: {
+            'application/json': {
+              schema: {
+                type: 'object',
+                required: ['token', 'amount', 'address'],
+                properties: {
+                  token: {
+                    type: 'string',
+                    description: 'UID of the token to mint.'
+                  },
+                  amount: {
+                    type: 'integer',
+                    description: 'The amount of tokens to mint. It must be an integer with the value in cents, i.e., 123 means 1.23.'
+                  },
+                  address: {
+                    type: 'string',
+                    description: 'Optional destination address of the minted tokens.'
```

Comment:
This is not optional, it's in the required array

Reply:
No. It is optional. I changed the required definition.

Reply:
See: https://github.com/HathorNetwork/hathor-wallet-headless/blob/5b82047d42409679ce57bafe1547606c37701445/src/api-docs.js#L1314